src/pages/studies/index.js

```diff
@@ -2,9 +2,110 @@ import matter from 'gray-matter';
 import Link from 'next/link';
 
 export default function StudyIndex(props) {
-  return props.studies.map(s => {
-    return (<Link key={s.slug} href={`/studies/${s.slug}`}>{s.frontmatter.title}</Link>)
-  });
+
+
+  // return props.studies.map(s => {
+  //   return (<Link key={s.slug} href={`/studies/${s.slug}`}>{s.frontmatter.title}</Link>)
+  // });
```

Comment:
Kept this original code that generates a dynamic posts list just in case we want to go back to this in the future.

Reply:
added it in!